### PR TITLE
docs: Update documentation.

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [x] Documentation content changes
 - [ ] Other, please describe:

## Impact on version

- [ ] **Major** (Public API was modified.)
  - Public constructs (class, struct, delegate, enum, etc.) were removed or renamed.
  - Public members were removed or renamed.
  - Public method signatures were changed or renamed.
- [ ] **Minor** (Public API was extended.)
  - Public constructs, members, or overloads were added.
- [x] **Patch** (Public API was unchanged.)
  - A bug in behavior was fixed.
  - Documentation was changed.
- [ ] **None** (The library is unchanged.)
  - Only code under the `build` folder was changed.
  - Only code under the `.github` folder was changed.

## Checklist

Please check that your PR fulfills the following requirements:

- [x] Documentation has been added/updated.
- [ ] Automated Unit / Integration tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).
- [x] Your conventional commits are aligned with the **Impact on version** section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,6 +301,18 @@ private void ConfigureAuthenticationTokenHandler(IServiceCollection services)
 }
 ```
 
+And don't forget to add needed `DelegatingHandler` to your endpoints, it won't be done automatically.
```
Comment (vincentcastagna) [inline README.md]: Small suggestion of rewording that :) 

You will also need to provide a `DelegatingHandler` to your endpoints if you want to handle the access token logic through the `AuthenticationTokenHandler` provided.
```csharp
private static IServiceCollection AddMyEndpoint(this IServiceCollection services, IConfiguration configuration)
{
	return services
		.AddEndpoint<IMyEndpoint, MyEndpointMock>(configuration, "MyEndpoint", configure: b => b
			.AddHttpMessageHandler<AuthenticationTokenHandler<AuthenticationData>>()
		);
}
```

Comment (Soap-141) [reply README.md]: Better sentence than mine! I'll update it.

**Review by vincentcastagna — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -301,6 +301,18 @@ private void ConfigureAuthenticationTokenHandler(IServiceCollection services)
 }
 ```
 
+You will also need to provide a `DelegatingHandler` to your endpoints if you want to handle the access token logic through the `AuthenticationTokenHandler` provided.
+
+```csharp
+private static IServiceCollection AddMyEndpoint(this IServiceCollection services, IConfiguration configuration)
+{
+	return services
+		.AddEndpoint<IMyEndpoint, MyEndpointMock>(configuration, "MyEndpoint", configure: b => b
```
Comment (jeanplevesque) [inline README.md:310]: This `AddEndpoint` method doesn't exists in this scope.

File: README.md [hunk 3]
```diff
@@ -301,6 +301,18 @@ private void ConfigureAuthenticationTokenHandler(IServiceCollection services)
 }
 ```
 
+You will also need to provide a `DelegatingHandler` to your endpoints if you want to handle the access token logic through the `AuthenticationTokenHandler` provided.
```
Comment (jeanplevesque) [inline README.md:304]: This isn't specific to `AuthenticationtokenHandler`. All the DelegatingHandlers shown in this page need to be added in some way to the desired `HttpClient`.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jeanplevesque) [reply README.md:310]: It's something custom to UnoApplicationTemplate and doesn't make sense in this context.

Comment (Soap-141) [reply README.md:310]: Yeah that's right it was as an example but by resolving an issue I'm getting a second one. I'll fix this.

**Pull request closed without merging**